Content.Shared/_DV/CosmicCult/Components/Examine/CosmicMarkLapseComponent.cs

```diff
@@ -0,0 +1,10 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._DV.CosmicCult.Components.Examine;
+
+/// <summary>
+/// Marker component for targets under the effect of Abberant Lapse.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class CosmicMarkLapseComponent : Component
+{ }
```

Comment:
If the `Mark` in here means `Marker`, it's superfluous
```suggestion
public sealed partial class CosmicLapseComponent : Component;
```